### ISSUE TITLE
Google discontinues "Allow Less Secure Apps"

### ISSUE THREAD

**Issue by mcnoat** [huggingface/knockknock#75]
"To help keep your account secure, starting May 30, 2022, ​​Google will no longer support the use of third-party apps or devices which ask you to sign in to your Google Account using only your username and password." -- https://support.google.com/accounts/answer/6010255?hl=en

Will that affect knockknock's mail notification feature?

**Comment by philipph77** [comment]
You are right, it is no longer possible to activate "access through less secure apps" in your google account. However, **you can still use knockknock's email sender by creating an app password in your google account** (see: [Sing in with App Passwords](https://support.google.com/mail/answer/185833?hl). Just create an app password for your script, and if asked enter that app password as mail password. Note that App Passwords can only be used with accounts that have [2-Step Verification](https://support.google.com/accounts/answer/185839) turned on.

**Comment by chiffa** [comment]
Which kinda defeats the purpose of creating a separate email account to manage the alerts.
Yagmail now has a pretty trivial Oauth2 implementation, so a patch is straightforward, but I am not seeing any pull requests accepted recently. 
Is the project still maintained?
